main.ledger

```diff
@@ -6873,7 +6927,265 @@
     Liabilities:Reimbursement:Zach Latta
     ; Receipt: 334493b534671849bf3e1dbfc4d9bba6.pdf
 
+2017/12/21 Bradfield
+    Expenses:Operating:Office:Rent                   $1,200.00
+    Assets:Chase:Checking
+    ; Receipt:
+
 2017/12/25 Pingdom
     Expenses:Operating:Software                         $14.95
     Liabilities:Reimbursement:Zach Latta
     ; Receipt: b2a230a8c90e034edc8f5f776705c149.pdf
+
+2017/12/26 Gusto
+    Expenses:Operating:Staff:Salary                  $3,131.84
+    Expenses:Operating:Staff:Salary                  $1,314.16
+    Assets:Chase:Checking
+    ; Receipt:
```

Comment:
Needs to be in correct payroll format - payee is not Gusto, they're just the bank transfer provider.